ignite/handlers/param_scheduler.py

```diff
@@ -313,6 +318,20 @@ def __init__(
         self.cycle = 0
         self.start_value_mult = start_value_mult
         self.end_value_mult = end_value_mult
+        self.warmup_each_cycle = warmup_each_cycle
+        if not self.warmup_each_cycle:
+            if warmup_duration is not None:
+                warnings.warn(
```

Comment:
@sihyeong671 actually why we should warn here? For me these two args `warmup_duration` and `warmup_each_cycle` are less tightly coupled:
```
warmup_duration is None
warmup_each_cycle can be True or False
=> no warm-up at all and nothing to warn about

warmup_duration is not None
warmup_each_cycle can be True or False
=> add a warm-up
```
What do you think ?

Reply:
Now I think `warmup_each_cycle` variable is not necessary. I will remove `warmup_each_cycle`!